src/libraries/System.Text.Json/gen/Reflection/RoslynExtensions.cs

```diff
@@ -37,7 +37,7 @@ public static MethodAttributes GetMethodAttributes(this IMethodSymbol methodSymb
 
             if (methodSymbol.IsAbstract)
             {
-                attributes |= MethodAttributes.Abstract;
+                attributes |= MethodAttributes.Abstract | MethodAttributes.Virtual;
```

Comment:
LGTM, and is also more consistent with reflection:
https://sharplab.io/#v2:EYLgtghglgdgNAFxAJwK7wCYgNQB8ACATAIwCwAUPgAwAE+xAdAJIDyA3BRfQJwAUCATwAOAUwD2AM14BlBMhEQwASgYBxEQgCyGgBZiMvAEQAxADaoAzjsMqAggjlRgqBCItK2QA===